client/src/components/layouts/Booking/Booking.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react';
+import InfiniteCalendar from 'react-infinite-calendar';
+import 'react-infinite-calendar/styles.css';
+import { Box } from '@material-ui/core';
+import RadioButton from '../BeforeVisit/Radio';
+
+const today = new Date();
+const lastWeek = new Date(
+  today.getFullYear(),
+  today.getMonth(),
+  today.getDate() - 7,
+);
+
+const Booking = (props) => {
+  const [selectedDate, setSelectedDate] = useState('');
+
+  const {
+    formField: { appointmentTime },
+  } = props;
+
+  const timeSlots = [
+    {
+      value: `${selectedDate} 10:00`,
+      label: '10:00 - 12:00',
+    },
+    {
+      value: `${selectedDate} 12:00`,
+      label: '12:00 - 14:00',
+    },
+    {
+      value: `${selectedDate} 14:00`,
+      label: '14:00 - 16:00',
+    },
+    {
+      value: `${selectedDate} 16:00`,
+      label: '16:00 - 18:00',
+    },
+  ];
+
+  const displayTimeSlots = (date) => {
+    setSelectedDate(
+      `${date.getFullYear()}-${date.getMonth() + 1}-${date.getDate()}`,
+    );
+  };
+  return (
+    <Box display="flex" flexDirection="column">
+      <InfiniteCalendar
+        width={400}
+        height={300}
+        selected={today}
+        disabledDays={[0, 6]}
+        minDate={lastWeek}
```

Comment:
some helpful comment for that dude too :grin:  `minDate` what's the use of lastWeek here

Reply:
This will be changed when we finish working on the calendar. That's why the pull request is not ready for review yet. 😅 But sure I will add many comments later.😜

Reply:
we know we know, that why we insist to review you anyway.. just to keep them in mind :smirk_cat:

Reply:
Doesn't matter draft or not.
we see a PR..we review it anyway :smirk_cat: